src/util-logopenfile.h

```diff
@@ -118,6 +118,9 @@ typedef struct LogFileCtx_ {
      * allow for rotataion. */
     uint8_t is_regular;
 
+    /* JSON flags */
+    size_t json_flags;
```

Comment:
Because that is what the API reference for libjansson specified for the flags parameter for json_dump_callback(). Do you want me to change it to something else?